keyboards/slothie60/keyboard.json

```
@@ -0,0 +1,263 @@
+{
+    "manufacturer": "makkomise",
+    "keyboard_name": "slothie60",
+    "maintainer": "makkomise",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+		"rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["A10", "B11", "B10", "B2", "B1", "B0", "A7", "A6", "A5", "A4", "A3", "A2", "A1", "A0"],
+        "rows": ["A8", "B15", "B14", "B13", "B12"]
+    },
+    "processor": "STM32F072",
+	"rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 14
+    },
+    "url": "https://github.com/makkomise/slothie60",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x7362"
+	},
+	"ws2812": {
+        "pin": "A14"
+    },    
+	"community_layouts": [
+		"60_iso"
+	],
+    "layouts": {
+		"LAYOUT_60_iso": {
+            "layout": [
+                {"x":0, "y":0},
```

All key entries in all layouts need the matrix positions like this:
```suggestion
                {"matrix": [0, 0], "x": 0, "y": 0},
```
(In theory some part of the build process could notice that there is a key with the exact same physical coordinates and sizes in another layout and copy `matrix` from that key, but this is currently not implemented, and you would need to provide the matrix positions for any non-matching keys anyway.)

---

> All key entries in all layouts need the matrix positions like this:
> 
> (In theory some part of the build process could notice that there is a key with the exact same physical coordinates and sizes in another layout and copy `matrix` from that key, but this is currently not implemented, and you would need to provide the matrix positions for any non-matching keys anyway.)


OHHHH! I missed that completely, sorry i think i'm just blind. Thank you so much for the help again! 
Now the lint check passed and it compiled without any errors, let me know if that's good now, thanks.